src/coreclr/gc/gc.cpp

```diff
@@ -28305,6 +28785,7 @@ void gc_heap::plan_phase (int condemned_gen_number)
 #endif //!USE_REGIONS
 
 #ifdef MULTIPLE_HEAPS
+
             //join all threads to make sure they are synchronized
             dprintf(3, ("Restarting after Promotion granted"));
             gc_t_join.restart();
```

Comment:
nit - seems to be a random empty line here

Reply:
Fixed.